config/base/software.nix

```
@@ -21,43 +21,44 @@ in
 
   environment.systemPackages = with pkgs; [
     # external package formats
-    flatpak
+    flatpak                      # Good software utility to use if you can't find what you want at nixpkgs
     # snapd N/A yet
 
     # sys apps
-    conf-tool
-    nixNodePackage
+    conf-tool                    # Config Tool
+    nixNodePackage               # NodeJS pre-installed
 
     # utilities
-    git
-    curl
-    wget
-    gnome3.gnome-disk-utility
+    git                          # That's git everybody, our favourite scm
+    curl                         # Ancient way of transfering data
```

wget is the more ancient one, actually

---

curl is mostly for devs, wget is mostly for people who want to download stuff
```suggestion
    curl                         # HTTP-debugger
```